### PR TITLE
Include further description of command line values

### PR DESCRIPTION
**Pull request by orangegrove1955**
# Description
I have included further descriptions of the values in the command line to more easily understand and follow these setup instructions

# Reasons
When attempting to follow these instructions to set up a new namespace for the first time, the arguments were unclear, especially considering these are getting started docs. It would be helpful to include this brief description for first time authors to understand what they need to input to the command line

### CODE REVIEW

File: jekyll/_cci2/orb-author-intro.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Enter the following command to claim your namespace, if you have not yet claimed
 circleci namespace create <name> <vcs-type> <org-name> [flags]
 ```
 
+where `name` is the namespace you wish to claim, `vcs-type` is the type of your version control system (i.e. `github` or `bitbucket`), and `org-name` is the name of your organisation
```
Comment (teesloane) [inline jekyll/_cci2/orb-author-intro.md:60]:
```suggestion
where `name` is the namespace you wish to claim, `vcs-type` is the type of your version control system (i.e. `github` or `bitbucket`), and `org-name` is the name of your organization.
```

Comment (teesloane) [reply jekyll/_cci2/orb-author-intro.md:60]: Good call. We should have this information here. I have made one small suggestion which you can commit using GitHub's interface; then we will be all set to merge.

Thank you for contributing!

**Review by teesloane — CHANGES_REQUESTED** [review]
G

**Comment by orangegrove1955** [comment]
@teesloane Thanks for the suggestion, I have committed it and believe this PR is ready to merge

**Review by teesloane — APPROVED** [review]

**Pull request merged**